### PR TITLE
Added a release script that automates the release process.

### PR DESCRIPTION
**Pull request by hategan**
It commits the RELEASE file, adds a tag, and builds the docker image.

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+git push
+
+git tag "v$VERSION"
+
+git push origin "v$VERSION"
+
+pushd deployment/docker
+
+read -p "Build? " BUILDING
```
Comment (andre-merzky) [inline release.sh:24]: What's the purpose of the read?  I don't see `$BUILDING` used anywhere.  Also, `build.sh` has it's own interactive check.

**Review by andre-merzky — APPROVED** [review]
Looks good to me, but left one comment for clarification.

BTW, unrelated to the PR: config.sh sets 
```sh
# Docker image to use
IMAGE=hategan/psi-j-testing-service
```

should that maybe move into the exaworks namespace (I think that exists and is used for the SDK)?

**Comment by hategan** [comment]
> should that maybe move into the exaworks namespace (I think that exists and is used for the SDK)?

Or maybe even a psi-j namespace?

File: release.sh (hunk 1 above)
Comment (hategan) [reply release.sh:24]: As usual, your reviews are invaluable. I was trying to debug something. Will remove.

**Comment by andre-merzky** [comment]
> > should that maybe move into the exaworks namespace (I think that exists and is used for the SDK)?
> 
> Or maybe even a psi-j namespace?

Yeah, that would also make sense of course once we release.  Until then anything goes, this should not block the PR.

**Pull request merged**